### PR TITLE
get_sites() fixes

### PR DESCRIPTION
**Pull request by jread-usgs**
for #98 and API needs

### CODE REVIEW

**Comment by jread-usgs** [comment]
@aappling-usgs review?

File: R/get_sites.R [hunk 1]
```diff
@@ -11,12 +11,13 @@
...
 #' @return a character vector of "site_root" titles (keys)
 #' @import sbtools
+#' @keywords internal
 #' @examples
 #' \dontrun{
 #' get_sites()
 #' get_sites(limit = 10)
 #' # get those sites that have water temperature
-#' get_sites(with_var_src = 'ts_wtr')
+#' get_sites(with_var_src = 'ts_wtr_nwis')
```
Comment (appling) [inline R/get_sites.R]: i wonder if we should change `get_sites` to accept a true `var_src` and turn it into a `ts_name` internally. or change `with_var_src` to `with_ts_name`. either could be appropriate, seeing that `get_sites` is an internal function, though i think the former will be easier to use within list_sites.

Comment (jread-usgs) [inline R/get_sites.R]: that would be better I think. create an issue?

Comment (aappling-usgs) [inline R/get_sites.R]: will do.

**Pull request merged**